test/e2e/specs/site-editor/site-editor-export.spec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site Editor Templates Export', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'clicking export should download emptytheme.zip file', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
+		await admin.visitSiteEditor( {
+			postId: 'emptytheme//index',
+			postType: 'wp_template',
+		} );
+		await editor.canvas.locator( 'body' ).click();
+		await page
+			.getByRole( 'region', { name: 'Editor top bar' } )
+			.getByRole( 'button', { name: 'Options' } )
+			.click();
+
+		const promise = page.waitForEvent( 'download' );
+		await page.getByRole( 'menuitem', { name: 'Export' } ).click();
+		const download = await promise;
+		expect( download.suggestedFilename() ).toBe( 'emptytheme.zip' );
```

Comment:
I really like how easy it's to test downloads with PW.

Reply:
💯, I love it.